test/unit/config/global-mocks.js

```diff
@@ -44,3 +45,6 @@ if ( ! global.TextDecoder ) {
 if ( ! global.TextEncoder ) {
 	global.TextEncoder = TextEncoder;
 }
+
+global.Blob = BlobPolyfill;
+global.File = FilePolyfill;
```

Comment:
Can these be set conditionally like TextDecoder and TextEncoder above?

```suggestion
if ( ! global.Blob ) {
	global.Blob = BlobPolyfill;
}
if ( ! global.File ) {
	global.File = FilePolyfill;
}
```

Reply:
I don't think so, because this overrides the default `Blob` implementation provided by jsdom, which is incomplete. Node on the other hand has the full thing already baked in.

Conditionally adding them causes tests to fail again.